content/blog/2023/sqlalchemy-for-django-developers.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: SQLAlchemy for Django Developers
 author: ed
-date: 2023-10-24
+date: 2023-10-20
```

Comment:
Is there a benefit to spacing these out just a bit, and publishing this article next week?

Reply:
Pushed it to Monday on 822f7c0d1909275b8ce25cccf7851f82a2eb72d0